bedrock/firefox/templates/firefox/new.html

```diff
@@ -155,7 +155,7 @@ <h1 class="large" id="main-headline">
     <div id="scene2-main" class="scene-content scene2 hidden">
     {% if l10n_has_tag('firefox_new_mobile_promo') %}
       <p class="help">
-        {{_('Your download should begin automatically. If not, <a id="%s" href="">click here</a>. It could take a few minutes, but it’s worth the wait.')|format('direct-download-link')}}
+        {{_('Your download should begin automatically. If not, <a id="%s" href="" data-link-type="download" data-download-os="Desktop">click here</a>. It could take a few minutes, but it’s worth the wait.')|format('direct-download-link')}}
```

Comment:
Because this link occurs in the middle of a string, adding in data attributes here breaks the translation for over over 100 locales. I suggest we avoid making changes like this in general, so we don't make extra work for our translators. Is there some other way you can track this link click?